src/main/java/com/google/sps/util/Validator.java

```diff
@@ -11,4 +11,14 @@ public class Validator {
     public static boolean validUserName(String username) {
         return username.matches("[\\w\\d]*");
     }
+
+    public static boolean userNameAvalible(String username){
+        try {
+            long userID = DataStoreHelper.queryUserID(username);
+        } catch (IllegalArgumentException e) {
+            return true;
```

Comment:
Is line 19 supposed to be returned in the try block?

Reply:
No. Since this function is searching for an available username I want the query to return IllegalArgumentException indicating that the username isn't in the database and by extension is available to sign up with.
Is there anything I should do to make that clearer?